lib/rules/jsx-sort-props.js

```diff
@@ -72,7 +72,7 @@ function contextCompare(a, b, options) {
     aProp = aProp.toLowerCase();
     bProp = bProp.toLowerCase();
   }
-  return aProp.localeCompare(bProp);
```

Comment:
1. this needs to return `0` when `aProp === bProp`.
1. i would still prefer to use localeCompare when case is not ignored.

Reply:
I will try to fix it with `localeCompare `.